squad/core/migrations/0131_create_resultsubmitters_auth_group_and_add_users.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 2.2.13 on 2020-07-06 15:28
+
+from __future__ import unicode_literals
+from django.db import migrations
+import re
+
+PERMISSIONS = ['view', 'add', 'change', 'delete']
+MODELS = {'core': ['Annotation',
+                   'Attachment',
+                   'Build',
+                   'Environment',
+                   'Metric',
+                   'Status',
+                   'Suite',
+                   'TestRun'],
+          'ci': ['TestJob']}
+
+
+def split_on_upper_and_make_lower(name):
+    split = re.findall('[A-Z][^A-Z]*', name)
+    return [w.lower() for w in split]
+
+
+def create_result_submitters_group_and_add_users(apps, schema_editor):
+    Group = apps.get_model('auth', 'Group')
+    Permission = apps.get_model('auth', 'Permission')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    result_sumbitters, created = Group.objects.get_or_create(name='result_submitters')
```

Comment:
please change the name of the group to "squad"

Reply:
@mwasilew ok, but why? isn't result_submitters bit more meaningful? or you mean that since this will apply to authenticated users, then squad is a more suitable name?

Reply:
IMHO result_submitters is misleading. Creating Annotation is not 'submitting results' for example. The 'squad' group would include all basic django permissions you need to work with SQUAD.

Reply:
yup..makes sense.